src/compiler/checker.ts

```diff
@@ -36331,6 +36351,7 @@ namespace ts {
             case SyntaxKind.ImportClause: // For default import
             case SyntaxKind.ImportEqualsDeclaration:
             case SyntaxKind.NamespaceImport:
+            case SyntaxKind.NamespaceExport:

```

Comment:
I don't think this is correct - namespace exports don't import anything locally.